reactioncommerce/reaction-admin
# fix: use updateProductVariantPrices mutation to save variant price

Signed-off-by: trojanh <rajan.tiwari@kiprosh.com>

Resolves https://github.com/reactioncommerce/reaction-admin/issues/170
Impact: **breaking|major**  
Type: **bugfix**

## Issue
Unable to set product variant price
1. Create a new variant.
2. Add a price to it
3. Refresh to see that the price is reset to zero.

## Solution
Use  `updateProductVariantPrices` mutation to set the price.

## Breaking changes
none

## Testing
1. Create a new variant.
2. Add a price to it
3. Refresh to see that the price is NOT reset to zero.
4. Also publish and refresh the variant to see it the changes get saved.

![variant-price-update](https://user-images.githubusercontent.com/22974490/71871561-873f6180-313f-11ea-8fd8-b1e03f8dcee9.gif)
